src/PullRequestReleaseNotes/Providers/GitHubPullRequestProvider.cs

```diff
@@ -40,28 +40,28 @@ public PullRequestDto Get(string commitMessage)
             var pullRequestId = ExtractPullRequestNumber(commitMessage);
             if (pullRequestId == null)
                 return null;
-            var pullRequest = _gitHubClient.PullRequest.Get(_programArgs.GitHubOwner, _programArgs.GitHubRepository, (int) pullRequestId).Result;
-            if (pullRequest == null)
-                return null;
-            if (_programArgs.VerboseOutput)
-                Console.WriteLine($"Found #{pullRequest.Number}: {pullRequest.Title}");
-            return GetPullRequestWithLabels(pullRequest);
+            return GetPullRequestWithLabels(pullRequestId.Value);
```

Comment:
I think we can gain a good performance boost by using Octokit in an async way rather than synchronously. Do you want to help me look at introducing use of async in the PullRequestProviders?